site/docs/master/csi.md

```diff
@@ -27,3 +27,6 @@ This section documents some of the choices made during implementation of the Vel
 
 Velero's support level for CSI volume snapshotting will follow upstream Kubernetes support for it, and will reach general availability sometime
 after volume snapshotting is GA in upstream Kubernetes. Beta support is expected to launch in Velero v1.4.
+
+
+[1]: customize-installation.md#enable-features#enable-server-side-features
```

Comment:
not sure which section you wanted to link to - you got both :D

Reply:
Still need to resolve this

Reply:
forgot to push 🤦